modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java

```diff
@@ -694,6 +701,10 @@ protected void validateFieldWithDDMFormFieldValidation(
 			return;
 		}
 
+		if (isHideField(ddmFormEvaluatorFieldContextKey)) {
+			return;
+		}
+
```

Comment:
@brunofariasdeo If the order doesn't matter, please sort it alphabetically, i.e., place this code after  `if (!isFieldVisible(ddmFormEvaluatorFieldContextKey)) {`.

In addition, how about creating a test for this? You can use the `DDMFormEvaluatorHelperTest.testRequiredValidationWithHiddenField` test as inspiration.